### ISSUE TITLE
Add CMMN and DMN viewer

### ISSUE THREAD

**Issue by pinussilvestrus** [pinussilvestrus/vs-code-bpmn-io#11]
Since we are able to simply render BPMN diagrams right now, it would be easily possible to extend the current functionality via [`dmn-js`](https://github.com/bpmn-io/dmn-js) and [`cmmn-js`](https://github.com/bpmn-io/cmmn-js/).

**Comment by salaboy** [comment]
Before adding these.. can we add the properties panel?

**Comment by pinussilvestrus** [comment]
Jup, but that would depend on #2. I'll create an issue for that.

**Comment by salaboy** [comment]
Do you want me to label issues with priorities.. ??

**Comment by pinussilvestrus** [comment]
Would be nice 👍 You can do that via milestones if you like to, as I did it in https://github.com/pinussilvestrus/vs-code-bpmn-io/issues/7

**Comment by bertisLongfellow** [comment]
Is development active, or deferred? I note the comments are 2019, the releases are 2020.

**Comment by budney** [comment]
I'm very interested in this as well. I have no experience writing VSCode extensions, or I'd try my hand at adding DMN support.
